apicapi/apic_client.py

```diff
@@ -403,63 +420,75 @@ def _send(self, request, url, data=None, refreshed=None, accepted=None,
                                          err_text=err_text, err_code=err_code)
         return imdata
 
+    def _get_request_call(self, request):
+        if request == ApicSession.HTTP_GET:
+            return self.session.get
+        elif request == ApicSession.HTTP_POST:
+            return self.session.post
+        elif request == ApicSession.HTTP_DELETE:
+            return self.session.delete
+        elif request == ApicSession.HTTP_PUT:
+            return self.session.put
+        else:
+            return None
```

Comment:
I don't know if it is possible to convert the session-method objects to the corresponding HTTP method string names without maintaining a mapping like this. If you can think of a better way, I'm all ears :)